assets/js/search-data.json

```diff
@@ -1,12 +1,12 @@
 ---
 ---
 {
-  {% for page in site.html_pages %}"{{ forloop.index0 }}": {
+  {% for page in site.html_pages %}{% if page.search_exclude != true %}"{{ forloop.index0 }}": {
     "id": "{{ forloop.index0 }}",
-    "title": "{{ page.title | xml_escape }}",
-    "content": "{{ page.content | markdownify | strip_html | xml_escape | remove: 'Table of contents' | strip_newlines | replace: '\', ' ' }}",
-    "url": "{{ page.url | absolute_url | xml_escape }}",
-    "relUrl": "{{ page.url | xml_escape }}"
-  }{% if forloop.last %}{% else %},
+    "title": "{{ page.title | replace: '&amp;', '&' }}",
+    "content": "{{ page.content | markdownify | strip_html | escape_once | remove: 'Table of contents' | remove: '```'  | remove: '---' | replace: '\', ' ' | normalize_whitespace }}",
```

Comment:
i'm assuming `normalize_whitespace` covers `strip_newlines. if not, then i think we should add that back.

Reply:
Yes, a little investigation indicates that normalize_whitespace covers the strip_newlines.  I got some of these changes from the main just-the-docs but we have to have our own version to take out some extra special characters.  At this point that is '[' and ']'.